src/templates/post.jsx

```diff
@@ -18,6 +19,7 @@ export default class PostTemplate extends React.Component {
     if (!post.category_id) {
       post.category_id = config.postDefaultCategoryID;
     }
+    const date = postNode.fields.date
```

Comment:
Alternatively, we can add this to `frontmatter` and reference it using `post.date`... which option would you prefer @skaldarnar ?

Reply:
this is fine :+1: